quora-service/src/main/java/com/upgrad/quora/service/entity/UserAuthTokenEntity.java

```diff
@@ -0,0 +1,122 @@
+package com.upgrad.quora.service.entity;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "USER_AUTH")
+@NamedQueries({
+        @NamedQuery(name = "userAuthTokenByAccessToken", query = "select ut from UserAuthTokenEntity ut where ut.accessToken =:accessToken")
+})
+public class UserAuthTokenEntity implements Serializable {
+
+    @Id
+    @Column(name = "ID")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @ManyToOne
+    @JoinColumn(name = "USER_ID")
+    private UserEntity user;
+
+    @Column(name = "UUID")
+    @Size(max = 200)
```

Comment:
use @NOTNULL annotation for uuid as it is mention in quora.sql class

Reply:
Changes made.